core-web/libs/sdk/experiments/src/lib/shared/models.ts

```diff
@@ -7,10 +9,11 @@
  * @property {boolean} debug - Specifies whether to enable debug mode for the experiment.
  */
 export interface DotExperimentConfig {
-    'api-key': string;
+    apiKey: string;
     server: string;
     debug: boolean;
     trackPageView?: boolean;
+    redirectFn?: (url: string) => void;
```

Comment:
Add doc to each property, check the client.

Reply:
added